public/js/settings/settings.component.js

```diff
@@ -0,0 +1,78 @@
+import angular from 'angular';
+
+export default angular.module('helloGov')
+.component('settingsForm', {
+    template: require('./settings.html'),
```

Comment:
This is using a client side (Angular) rendered template. I'm taking advantage of webpack to handle bundling it up with the rest of the JS.

The reason I chose to render the whole template on the client is this way I don't have to maintain rendering on both the server (for the initial render) and the client (for subsequent changes when you update a setting).